Assets/Scripts/Controller/Pickup Pup/PPGameController.cs

```diff
@@ -115,6 +142,9 @@ protected override void fetchReferences()
 		dataController = PPDataController.GetInstance;
 		dataController.SetFilePath(SAVE_FILE_PATH);
 		dataController.LoadGame();
+
+        // Defaults not working for some reason. Help me fix!
```

Comment:
`getDefaultFile()` only gets called if a save does not already exist upon load. If you want the player's coins to be set to 2000 at the start of each session (temporarily for testing purposes), you could modify `public PPGameSave LoadGame()` which is also in `PPDataController`